roadmap.md

```diff
@@ -0,0 +1,35 @@
+Roadmap Zombienet v2
+
+## Infra
+- Chaos testing, add examples and explore possibilities in `native` and `podman` provider
+- Add `docker` provider
+- Add `nomad` provider
+- Create helm chart to allow other use zombienet in k8s
+- Auth system to not use k8s users
+- Create GitHub Action and publish in marketplace
+
+## Intenal teams
+- Add more teams (wip)
+
+## UI
+- Create UI to create `feature` and `network` file.
+- Create script to translate from polkadot launch config to zombienet network file.
+- Improve vscode extension (grammar/snippets)
+
+## Registry
+- Create decorators registry and allow override by paras (wip)
+- Explore how to get info from paras.
+
+## Functional tasks
```

Comment:
Zombienet Test SDK (Rust) - it's not replacing the deployment which remains as is (typescript) + additional features listed above.
- one language to fully write the entire test (deployment part + actual tests to be performed)
- simple APIs to generate the `Network` file - builder pattern
- simple APIs to query/assert metrics
- write more complex test logic
- directly use or wrap subxt - open HRMP channels, send XCM, do runtime upgrades, etc
- opens up the Rust ecosystem of libraries that can be used in a test 
- feature parity with current DSL support (phasing out DSL over time after we have same featureset)
- easier for people outside Parity to contribute to the project